galata/test/jupyterlab/styles.test.ts

```diff
@@ -0,0 +1,203 @@
+// Copyright (c) Jupyter Development Team.
+// Distributed under the terms of the Modified BSD License.
+
+import { expect, test } from '@jupyterlab/galata';
+
+// posssibly incomplete (as new tags get added) list of HTML tags as defined in HTML standards (including deprecated elements)
```

Comment:
```suggestion
// possibly incomplete (as new tags get added) list of HTML tags as defined in HTML standards (including deprecated elements)
```

Do you think it would be possible to use something like https://github.com/kimamula/ts-transformer-keys to get it from `HTMLElementTagNameMap` and `HTMLElementDeprecatedTagNameMap` in lib.dom.d.ts?

Reply:
Yes, that would be neat; I will work on more tests for this test suite (e.g. to find potentially unused styles) and if I end up using this constant in other tests I can try the transformer approach in a follow-up.

I did consider adding a package from npm which lists the tags but decided against it to avoid increasing maintenance burden. My reasoning is that once the upstream issue in Chromium gets resolved we may be able to drop this test, so maybe we don't need to be future-proof here.